Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.cpp

```diff
@@ -659,6 +683,19 @@ void RemoteLayerTreeEventDispatcher::handleSyntheticWheelEvent(PageIdentifier pa
     internalHandleWheelEvent(platformWheelEvent, processingSteps);
 }
 
+void RemoteLayerTreeEventDispatcher::didStartRubberbanding()
+{
+    if (m_MomentumIntervalIsActive) {
+        WTFEndSignpostAlways(nullptr, ScrollingPerformanceTestMomentumInterval, "isAnimation=YES;");
```

Comment:
These two lines repeat in two places, making things error-prone to future edits. Consider a helper function for them and the FingerDown version.

Reply:
I added 4 helper functions for start/end fingerDown/momentum